public/database/movies.json

```diff
@@ -0,0 +1,244 @@
+{
+  "movies": [
+    {
+      "id": 1,
+      "title": "Psycho",
+      "story": "Sekreteraren Marion Crane har ledsnat på sitt gråa liv. Hon knycker företagets kassa och flyr med den för att möta sin älskare. På vägen dit tar hon in på ett ödsligt motell men den trevlige motellägaren Norman Bates.",
+      "genre": "Thriller",
+      "hour": "1",
+      "minute": "49",
+      "coverimage": "images/covers/psycho-cover.jpg",
+      "sliderimage": "images/sliders/psycho.jpg",
```

Comment:
I recommend adding descriptions for the photos to enhance accessibility and provide a fallback in case there are issues with uploading or displaying the images.

Reply:
Good idea! Where do i write that?